modules/cronjob_lotus_chain_snapshots/variables.tf

```diff
@@ -34,3 +34,9 @@ variable "blocks_to_export" {
   type    = number
   default = 900
 }
+
+variable "is_suspended" {
+  type = bool
+  default = false
+  description = "Set cronjob status to suspended if true"
+}
```

Comment:
Could you configure your IDE to put an empty string in the end of file?

Reply:
Done